plugins/ruby-foreman-remote-execution/debian/install

```diff
@@ -1,5 +1,5 @@
 foreman_remote_execution.rb           usr/share/foreman/bundler.d
 cache                                 usr/share/foreman/vendor
 assets                                var/lib/foreman/public
-webpack                               var/lib/foreman/public
+webpack/foreman_remote_execution      var/lib/foreman/public/webpack
```

Comment:
Interesting, I wasn't aware I'd have to change anything else than the versions

Reply:
See GH-4025 for the full story, I just wanted to take the opportunity to bring this in with the udpate.

Reply:
Ok, makes sense. Thanks for stepping in